ethernet_init/Kconfig.projbuild

```diff
@@ -62,6 +62,13 @@ menu "Ethernet Configuration"
                     The following chips are supported: KSZ8001, KSZ8021, KSZ8031, KSZ8041,
                         KSZ8051, KSZ8061, KSZ8081, KSZ8091
                     Goto https://www.microchip.com for more information about them.
+
+            config ETHERNET_PHY_LAN867X
+                bool "LAN867x"
+                help
+                    The LAN8670/1/2 is a high-performance 10BASE-T1S single-pair Ethernet PHY transceiver for 10 Mbit/s half-duplex networking over a single pair of conductors
+                    The following chips are supported: LAN8670, LAN8671
```

Comment:
Any reason why LAN8672 is not supported?

Reply:
LAN8672 is MII-only.

Reply:
It doesn't matter from PHY driver point of view. It's matter of EMAC configuration. Just remove "The following chips are supported: LAN8670, LAN8671"

Reply:
Please resolve the comment.

Reply:
Updated.